[UXPLATFORM-6046]Added reflowMultipleDataset documentation in panning docs. (#273)
cerner/terra-graphs: packages/terra-graphs-docs/src/terra-dev-site/graph/CoreConfiguration.b/Documentation/Panning.12.graph.mdx

};
```

### reflowMultipleDatasets

> Panning can only be used with graphs having the construct as Graph, Gantt and Timeline.

Reviewer: nit:
```suggestion
Panning can be used with all graph types except Pie.
```
